### DISCUSSION TITLE
Thrust to IDLE should switch of A/T

### DISCUSSION THREAD

**Discussion by mga010** [flybywiresim/aircraft#2435] [Help]
Title says it. Since I could not find a way to press the disconnect button on the thrust lever, I used the alternative method to retard to IDLE. But that does not work.

**Comment by MapleNAV-TCASFAULT** [comment]
What controls do you use for throttles?

**Comment by mga010** [comment]
I am using a Thrustmaster lever. But it does not matter for the question. I am looking to assign a key to the disconnect button of the A/T at the real lever.

**Reply by MapleNAV-TCASFAULT** [reply]
Well the A/THR should disconnect when the throttles are at the IDLE detent. Which means that there shouldn't be a need for an A/THR disconnect button.

**Reply by mga010** [reply]
The "need for an A/THR disconnect button" is not a matter of discussion. It is there in the real thing and used on almost every approach.

**Comment by MapleNAV-TCASFAULT** [comment]
Well you can assign that button my searching up "autothrottle disconnect". Its a toggle and it can and should work with that button on your throttle. If it doesn't then its a MSFS incompatibility issue, and you'll just have to use the FCU A/THR button instead.
